front/src/pages/nft-catalog.tsx

```diff
@@ -1,9 +1,27 @@
 import React from "react";
+import { useQuery } from "react-query";
+import { getAllNfts } from "../services/nftService";
+import { NFTMicroCard } from "../components/nft-micro-card";
+import { NFTCardDto } from "../schema/NFTCatalogSchema";
 
-type Props = {};
+const NFTCatalog = () => {
+    const { data, error, isLoading, isError } = useQuery("getAllNfts", getAllNfts);
```

Comment:
Can we extract this into `nftService.ts` until we decide how will we store global states?
We could have it as a hook there which just returns these these objects which you are destructuring here. In that way, if we change logic for storing global states it will be easier to do that, and reuse this state throughout the app.

Reply:
yes we can :)
Idk for now ho to implement this
Buti will redo, suggest implementation, and u will correct me :)